src/Search.elm

```diff
@@ -287,6 +287,11 @@ channelDetails channel =
         Release_20_09 ->
             ChannelDetails "20.09" "20.09" "nixos/release-20.09" "nixos-20.09"
 
+-- Concentrates logic near releases definition.
+-- Otherwise you would have remember to modify @:124 defaultChannel each time.
+-- Bump when new channel is released.
+latest_stable : String
+latest_stable = "20.09"
```

Comment:
I think we should do this more programmatically. It would be more work but also require less maintenance.